testsuite/features/step_definitions/common_steps.rb

```diff
@@ -1467,6 +1467,14 @@ def server_secret
   end
 end
 
+When(/^I reboot the "([^"]*)" minion through SSH$/) do |host|
+  node = get_target(host)
+  node.run('reboot > /dev/null 2> /dev/null &')
+  default_timeout = 120
```

Comment:
Can we call it reboot_timeout instead please?
I know we have the old step also using default_timeout... but... I don't think is good we use the default_timeout variable having another variable with exact same name but in upper-case

Reply:
Yes, I will adjust that in a moment.